### DISCUSSION TITLE
Handymen mowing was useless, how about ride on mowers and tractors with mower attachments?

### DISCUSSION THREAD

**Discussion by ThiaJay** [OpenRCT2/OpenRCT2#18190] [Feature requests]
It wasn't really realistic to have handymen mowing huge plots of land in the game. It would have made more sense to have a combination of handymen with strimmers and some ride on mowers/utility tractors with mower attachments for large areas of grass.

For an idea of how that looks in real life: https://www.deere.co.uk/en/commercial-mowing/

**Comment by 733737** [comment]
I don't think that this will ever be in the game officially but it is already possible with tractor custom vehicles and the tile inspector

**Comment by ocalhoun6** [comment]
Yeah ... that would take a *lot* of work to create. New code for how handymen operate, new sprites, etc. Even the surface sprites would have to be changed, because grass mowed by a big industrial mower would look different than the stripes you get with the handyman's small mower. (Heck, to really do it properly, I'd say it would have to be an entirely new staff type, like "Landscaper" or something. Because otherwise it's really weird when your handymen whip a huge industrial mower out of their pocket.)

And since lawn mowing doesn't serve any practical in-game purpose -- only aesthetics, I doubt it's going to be a feature that most players are going to be very interested in.

If you do want large areas of your park mowed without needing an army of handymen roaming over it, you could use cheats to mow all the grass and keep it mowed ... or you could use custom scenery objects to give that look. (Now that custom terrain objects can be imported, has anybody made a 'permanently mowed grass' surface object yet?)
